src/main/java/io/nats/client/NatsThread.java

```diff
@@ -33,7 +29,7 @@ public NatsThread(Runnable runnable, String name) {
         super(runnable, name + "-" + created.incrementAndGet());
         setUncaughtExceptionHandler(new Thread.UncaughtExceptionHandler() {
             public void uncaughtException(Thread t, Throwable e) {
-                logger.debug("UNCAUGHT in thread {}", t.getName(), e);
+               e.printStackTrace();
```

Comment:
The exception was swallowed/discarded to begin with, and I don't want to change behavior too much at this point.